### ISSUE TITLE
TypeError: undefined is not a function with version 6.3.3

### ISSUE THREAD

**Issue by Lucas-C** [stylelint/stylelint#1243]
This is a bug that appears in v6.3.3 (v6.3.2 is OK).

This may be related to the rule "unit-no-unknown" given the error message, but I'm unsure.

Here is the `dummy.css` file I tested:
```css
.dummy {
    width: 1px;
}
```

Here is the `.stylelintrc` file I used:
```json
{
  "extends": ["stylelint-config-standard"]
}
```

I'm using stylint CLI:
```
npm install --save-dev stylelint@6.3.3 stylelint-config-standard@6.0.0
$(npm bin)/stylelint dummy.css --config .stylelintrc
```

This issue is not related to non-standard syntax.

I expected a normal analysis report.

I got the following error output:
```shell
TypeError: undefined is not a function
    at _toConsumableArray (G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\stylelint\dist\utils\isKnownUnit.js:24:182)
    at exports.default (G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\stylelint\dist\utils\isKnownUnit.js:19:38)
    at G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\stylelint\dist\rules\unit-no-unknown\index.js:43:45
    at walk (G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\postcss-value-parser\lib\walk.js:7:22)
    at ValueParser.walk (G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\postcss-value-parser\lib\index.js:18:5)
    at G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\stylelint\dist\rules\unit-no-unknown\index.js:26:48
    at G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\postcss\lib\container.js:94:28
    at G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\postcss\lib\container.js:81:26
    at Rule.each (G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\postcss\lib\container.js:68:22)
    at Rule.walk (G:\devhome\tools\cygwin\home\lucas_cimon\stylelint-bug-reproduction\node_modules\postcss\lib\container.js:80:21)
```

**Comment by Lucas-C** [comment]
Note I'm using Cygwin
```shell
$ node --version
v0.12.2
$ npm --version
3.8.7
```

**Comment by evilebottnawi** [comment]
@Lucas-C see as bug in postcss

**Comment by evilebottnawi** [comment]
Reproduce on Ubuntu node v0.12.2 (use nvm)

**Comment by evilebottnawi** [comment]
WOrk fine on node 6(latest), 5(latest), 4(latest), bugs only on 0.12(any version)

**Comment by evilebottnawi** [comment]
@davidtheclark see as babel bug https://github.com/babel/babel/pull/3468
After transform we get:
```
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});

exports.default = function (unit) {
  var _ref = arguments.length <= 1 || arguments[1] === undefined ? {} : arguments[1];

  var _ref$only = _ref.only;
  var only = _ref$only === undefined ? false : _ref$only;

  var unitLowerCase = unit.toLowerCase();

  if (only && only === "length") {
    return knownLengthUnits.has(unitLowerCase);
  }

  console.log(knownNonLengthUnits); // <-- These variable is empty on node 0.12.*
  console.log(knownLengthUnits); // <-- These variable is empty on node 0.12.*
  process.exit()

  var knownUnits = new Set([].concat(_toConsumableArray(knownNonLengthUnits), _toConsumableArray(knownLengthUnits)));

  return knownUnits.has(unit) || knownUnits.has(unitLowerCase);
};

function _toConsumableArray(arr) { if (Array.isArray(arr)) { for (var i = 0, arr2 = Array(arr.length); i < arr.length; i++) { arr2[i] = arr[i]; } return arr2; } else { return Array.from(arr); } }

/**
 * Check is known unit
 *
 * @param {string} Unit
 * @param {object} args - Named arguments object
 * @param {string} [args.only=false] - Check unit only with length
 * @return {boolean} If `true`, the unit is known
 */

var knownNonLengthUnits = new Set([
// Relative length units
"%",
// Time length units
"s", "ms",
// Angle
"deg", "grad", "turn", "rad",
// Frequency
"Hz", "kHz",
// Resolution
"dpi", "dpcm", "dppx"]);

var knownLengthUnits = new Set([
// Relative length units
"em", "ex", "ch", "rem",
// Viewport-percentage lengths
"vh", "vw", "vmin", "vmax", "vm",
// Absolute length units
"px", "mm", "cm", "in", "pt", "pc", "q"]);
```
